catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -195,9 +196,10 @@
       new NitfHeaderAttribute(
           Security.CLASSIFICATION_SYSTEM,
           "FSCLSY",
-          header -> header.getFileSecurityMetadata().getSecurityClassificationSystem(),
-          new SecurityAttributes().getAttributeDescriptor(Security.CLASSIFICATION_SYSTEM),
-          FILE_CLASSIFICATON_SECURITY_SYSTEM);
+          header ->
+              NitfUtilities.fipsToSingleIsoOrException(
+                  header.getFileSecurityMetadata().getSecurityClassificationSystem()),
+          new SecurityAttributes().getAttributeDescriptor(Security.CLASSIFICATION_SYSTEM));
```

Comment:
❓ Looks like we dropped the last `FILE_CLASSIFICATON_SECURITY_SYSTEM`. Was that on purpose?